tests/impl.cpp

```diff
@@ -1525,6 +1576,114 @@ bool test_mm_cmplt_ps(const float *_a, const float *_b)
     return validateInt32(iret, result[0], result[1], result[2], result[3]);
 }
 
+bool test_mm_cmplt_ss(const float *_a, const float *_b)
+{
+    __m128 a = test_mm_load_ps(_a);
+    __m128 b = test_mm_load_ps(_b);
+
+    float result[4];
+    result[0] = _a[0] < _b[0] ? getNAN() : 0;
+    result[1] = _a[1];
+    result[2] = _a[2];
+    result[3] = _a[3];
+
+    __m128 ret = _mm_cmplt_ss(a, b);
+    return validateFloat(ret, result[0], result[1], result[2], result[3]);
+}
+
+bool test_mm_cmpnlt_ps(const float *_a, const float *_b)
+{
+    __m128 a = test_mm_load_ps(_a);
+    __m128 b = test_mm_load_ps(_b);
+
+    int32_t result[4];
+    result[0] = !(_a[0] < _b[0]) ? -1 : 0;
+    result[1] = !(_a[1] < _b[1]) ? -1 : 0;
+    result[2] = !(_a[2] < _b[2]) ? -1 : 0;
+    result[3] = !(_a[3] < _b[3]) ? -1 : 0;
+
+    __m128 ret = _mm_cmpnlt_ps(a, b);
+    __m128i iret = *(const __m128i *) &ret;
+    return validateInt32(iret, result[0], result[1], result[2], result[3]);
```

Comment:
Why don't you write the test like `test_mm_cmpnlt_ss` does?

Reply:
I think the way you write in `test_mm_cmpnlt_ss` is better than the original test written style.